Cql/Cql.Compiler/CqlOperatorsBinding.Bind.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Hl7.Cql.Operators;
+using Hl7.Cql.Primitives;
+
+namespace Hl7.Cql.Compiler;
+
+#pragma warning disable CS1591
+partial class CqlOperatorsBinding
+{
+    private static readonly IReadOnlyDictionary<string, MethodInfo[]> ICqlOperators_MethodInfos_By_Name =
+        typeof(ICqlOperators)
+        .GetMethods(BindingFlags.Instance | BindingFlags.Public)
+        .GroupBy(m => m.Name)
+        .ToDictionary(m => m.Key, m => m.ToArray());
+
+    private static readonly ConstantExpression NullObject_ConstantExpression = Expression.Constant(null, typeof(object));
+
+    private MethodCallExpression? BindToMethodConvertArgsOrNull(
```

Comment:
Would you mind adding a bit of comments to document what this method is trying to achieve?

Reply:
Comments added. Also broke up this method into two - one for resolving the method info and converted arguments, and the other for creating the call expression